pages/index.js

```diff
@@ -8,11 +8,17 @@ import Link from "next/link";
 import SVGIcon from "@/components/Icons/SVGIcon";
 import SearchForm from "@/components/SearchForm";
 import StyledMessage from "@/components/List/Message/StyledMessage";
-import StyledTitle from "@/components/Header/StyledTitle";
+import TranslationForm from "@/components/TranslationForm";
+import AddButton from "@/components/Buttons/AddButton";
+import SearchButton from "@/components/Buttons/SearchButton";
+import TranslateButton from "@/components/Buttons/TranslateButton";
 
 export default function HomePage() {
   const [translationList, setTranslationList] = useAtom(translationListAtom);
   const [isFound, setIsFound] = useState();
+  const [isAddMode, setIsAddMode] = useState(false);
+  const [isSearchMode, setIsSearchMode] = useState(false);
+  const [isTranslateMode, setIsTranslateMode] = useState(false);
```

Comment:
Are these modes independent from each other? In other words: can I open all Forms at the same time or should only be one form active at a time?